src/EditorFeatures/VisualBasicTest/Diagnostics/PreferFrameworkType/PreferFrameworkTypeTests_FixAllTests.vb

```diff
@@ -348,9 +346,9 @@ End Class]]>
                                    <Document><![CDATA[
 Imports System
 Class ProgramA
-    Dim x As Integer = 0
-    Dim y As Integer = 0
-    Dim z As Integer = 0
+    Dim x As Int32 = 0
+    Dim y As Int32 = 0
+    Dim z As Int32 = 0
```

Comment:
This behavior is what we want.  We've consolidated to a single ID, and so now we update all of these when someone does 'fix all'.